native/src/main/java/io/ballerina/persist/nodegenerator/SyntaxTreeConstants.java

```diff
@@ -0,0 +1,48 @@
+package io.ballerina.persist.nodegenerator;
```

Comment:
License header is missing

Reply:
Updated in [d6a53d3](https://github.com/ballerina-platform/persist-tools/pull/15/commits/d6a53d3f948c8f076acb3cc1f2730258a998cf40)